lib/galaxy/datatypes/sniff.py

```diff
@@ -359,7 +359,7 @@ def guess_ext(fname, sniff_order, is_binary=False):
     >>> fname = get_test_fname('3unsorted.bam')
     >>> guess_ext(fname, sniff_order)
     'unsorted.bam'
-    >>> fname = get_test_fname('test.idpDB')
+    >>> fname = get_test_fname('test.idpdb')
```

Comment:
This is the filename, which was correct before. Revert?

Reply:
No, this is intentional, to map the file extension to the datatype extension